libs/src/api/Track.ts

```diff
@@ -403,10 +436,9 @@ export class Track extends Base {
     this.IS_OBJECT(metadata)
     const ownerId = this.userStateManager.getCurrentUserId()
     if (!ownerId) {
-      return {
-        error: 'No users loaded for this wallet'
-      }
+      throw new Error('No users loaded for this wallet')
```

Comment:
just ensuring throwing here is chill compared to returning error object?

Reply:
yep yep - the flow already had other places that could throw